### ISSUE TITLE
[XAML Controls Gallery(debug)->BasicInput->ColorPicker] : Few of the drop down values are not visible in HC white, Black modes

### ISSUE THREAD

**Issue by v-amnat** [microsoft/Xaml-Controls-Gallery#681]
**Tags**
Accessibility, A11yCT, A11yMAS, A11yMediumImpact, MAS4.3.1, SupportingthePlatform,
 A11ySev2, XAMLControlsGallery, WinUI, Win32, Wipro

**Test Environment**
1. XAML Controls Gallery(debug)  version - 2.0.21.0
2. Win 32/ OS build- 21343.rs_Prerelease.210320-1757

**Steps to reproduce the bug**

1. Launch the XAML Controls Gallery (Debug) App
2. Go to the Basic Input-> ColorPicker.
3. Navigate to drop downs across the color picker tile and observe

**Actual behavior**

In HC White and black modes values in the drop down buttons are not visible.

Same issue is seen in below tiles
Color Picker
GridView
Splitview
Inkcanvas
Easingfunctions
Pipespager
ScrollViewer
InfoBar
RadialGradientBrush
Revealfocus 
RichTextBox
Sound
Acrylic
ListBox

**Expected**
All content on the page should remain visible/distinguishable in Windows high contrast modes. Typically, these problems are caused by using icons or images when CSS or text could be used, or by not using semantic code to build interactive components. This is a valuable resource for high contrast mode fixes. 

**User Impact** 
When content or components disappear in high contrast modes or they don’t appear as interactive elements, customers who rely on that feature are blocked from information and functionality of the full user experience within the site/app. 

**MAS Reference**
MAS 4.3.1 - No Disruption of Accessibility Features 

**Attachments**

[Xaml drop down values not visible.zip](https://github.com/microsoft/Xaml-Controls-Gallery/files/6310857/Xaml.drop.down.values.not.visible.zip)

**Comment by ranjeshj** [comment]
Fixed in latest WinUI bits.
![image](https://user-images.githubusercontent.com/28935693/116916500-41df5700-ac02-11eb-8c03-d3b5a73b3376.png)
